.github/workflows/rollback.yml

```
@@ -0,0 +1,76 @@
+    steps:
+      - name: Checkout deployment scripts
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: |
+            deploy
+
+      - name: Set up gcloud CLI
+        uses: google-github-actions/setup-gcloud@v2
+        with:
+          project_id: ${{ secrets.GCP_PROJECT_ID }}
+          service_account_key: ${{ secrets.GCP_SA_KEY }}
```

actionlint (1.7.7)

input "service_account_key" is not defined in action "google-github-actions/setup-gcloud@v2". available inputs are "install_components", "project_id", "skip_install", "version"

(action)